extensions/amp-lightbox-viewer/0.1/amp-lightbox-viewer.css

```diff
@@ -85,11 +85,42 @@
   background-color: rgba(0,0,0, 1) !important;
 }
 
+.-amp-lightbox-viewer-mask {
+  z-index: 2147483643 !important;
+}
+
+.-amp-lightbox-viewer-gallery-mask {
+  display: none !important;
+  z-index: 2147483646 !important;
+}
+
+.-amp-lightbox-viewer[gallery-view] .-amp-lightbox-viewer-gallery-mask {
+  display: block !important;
+}
+
+.-amp-lightbox-viewer-gallery-grid {
```

Comment:
if we go with the approach of a single  `.-amp-lightbox-viewer-gallery` that also acts as the mask, the this can just become

.-amp-lightbox-viewer-gallery {
    top: 50px!important;
}